walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,64 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.Collectors;
```

Comment:
I'm not sure the HashSet, Set, List, Arrays, and Collectors imports are used so you could probably remove them. Make sure you rerun the tests after you remove them though to ensure that there won't be any issues!